src/libraries/System.Runtime.Numerics/src/System/Numerics/BigInteger.cs

```diff
@@ -989,19 +989,18 @@ public override int GetHashCode()
 
             if (_bits == null)
                 return _sign;
-            int hash = _sign;
+
+            HashCode hash = default;
+            hash.Add(_sign);
             for (int iv = _bits.Length; --iv >= 0;)
-                hash = NumericsHelpers.CombineHash(hash, unchecked((int)_bits[iv]));
-            return hash;
+                hash.Add(_bits[iv]);
+            return hash.ToHashCode();
```

Comment:
Maybe add a // TODO: https://github.com/dotnet/runtime/issues/48702 comment to use that when it's available.

On that note, is there a reason this is iterating down rather than up?  If we're ok changing the hashcode to one that's randomized, we can't be too concerned about the order here, right?

Reply:
Probably not, @huoyaoyuan is this something you could fix (just so it goes from 0 to length)?

Reply:
https://github.com/dotnet/runtime/pull/54168 was merged.  I think this should now be able to be something like:
```C#
hash.AddBytes(MemoryMarshal.AsBytes(_bits));
```